flink-rpc/flink-rpc-akka/src/main/java/org/apache/flink/runtime/rpc/akka/AkkaRpcService.java

```diff
@@ -480,14 +488,13 @@ public ScheduledExecutor getScheduledExecutor() {
 
     @Override
     public void execute(Runnable runnable) {
-        actorSystem.dispatcher().execute(runnable);
+        internalExecutor.execute(runnable);
```

Comment:
would there be any downside in ditching this and just using the internalScheduledExecutor?